dashboard/app/models/lesson.rb

```diff
@@ -723,6 +723,88 @@ def lesson_plan_has_verified_resources
     resources.any? {|r| r.audience == 'Verified Teacher'}
   end
 
+  def self.copy_to_script(original_lesson, destination_script)
+    return if original_lesson.script == destination_script
+    raise 'Both lesson and script must be migrated' unless original_lesson.script.is_migrated? && destination_script.is_migrated?
+    raise 'Destination script must be in a course version' if destination_script.get_course_version.nil?
+
+    ActiveRecord::Base.transaction do
+      copied_lesson = original_lesson.dup
+      copied_lesson.key = copied_lesson.name
+      copied_lesson.script_id = destination_script.id
+      copied_lesson.lesson_group_id = destination_script.lesson_groups.last.id
+
+      copied_lesson.absolute_position = destination_script.lessons.count + 1
+      numbered_lesson = copied_lesson.has_lesson_plan || !copied_lesson.lockable
```

Comment:
You should be able to just make this line `copied_lesson.numbered_lesson?`

Reply:
Ahh thank you!